pkg/account/delete/client.go

```diff
@@ -50,23 +52,104 @@ func NewAccountAPIClient(accountUUID string, restClient *accounts.Client) Client
 	}
 }
 
-// NotFoundErr is a sentinel error signifying that the resource desired to be deleted was not found. Generally this error can be treated as a succeful "deletion" of the resource.
-var NotFoundErr = errors.New("nothing with given name found")
+// NotFoundError is an error signifying that the desired resource was not found. Generally this error can be treated as a successful "deletion" of the resource.
```

Comment:
Comment not matching struct name after renaming the struct

Reply:
Fixed 👍